src/post/ActionBar/index.js

```diff
@@ -11,11 +11,14 @@ import { getAuthUsername } from '../../reducers/authUserReducer';
 import Icon from '../../icons/Icon';
```

Comment:
There are a lot of formatting changes in this file. If something is allowed by the `eslint` and `editorconfig`, I don't think we need to change it for no reason.

Reply:
I have take care of formatting changes as well in the recent commit, kindly have a look there.